src/Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/common/GetFormatDataCommand.cs

```diff
@@ -4,6 +4,8 @@
 using System;
 using System.Linq;
 using System.Management.Automation;
+using System.Management.Automation.Internal;
+
```

Comment:
Please alphabetically resort all using-s.

Reply:
Isn't this already sorted?

Reply:
We have other using-s below.

It is minor comment.